.github/pull_request_template.md

```diff
@@ -1,32 +1,30 @@
-### Description
+Git commit description, explain the changes you made here
 
-Please briefly explain the changes you made here.
+---
+__*Leave above in PR description, copy the below into a comment*__
+___
```

Comment:
Someone will 100% leave the `Git commit description, explain the changes you made here` part in.

Reply:
Perfect is the enemy of good enough